### ISSUE TITLE
Error when including vue-async-computed from CDN

### ISSUE THREAD

**Issue by Alex-Sokolov** [foxbenjaminfox/vue-async-computed#40]
With 3.3.1 (no errors): http://jsfiddle.net/gnxvyqac/1/
With 3.4.0 (errors): http://jsfiddle.net/gnxvyqac/2/

```
Uncaught TypeError: Cannot set property 'AsyncComputed' of undefined
```

**Comment by foxbenjaminfox** [comment]
Yes, that's quite bad. It's fixed now in `v3.4.1`.

`v3.4.0` included a change in the build step from `babel-preset-2015` to `babel-preset-env`, which messed this up by adding a `"use strict";`, and then optimizing `rollup`s use of `this` in the global scope into `undefined`.

I've fixed this by [passing](https://github.com/foxbenjaminfox/vue-async-computed/commit/d2dc0349135428464fb876f65345a629a7046c8f) the appropriate `{ "modules": false }` option to `babel-preset-env`, which gets it to behave the way you'd expect in this case.

Thanks for pointing this issue out!

**Comment by Alex-Sokolov** [comment]
Thanks!
